cli/bench/types.go

```diff
@@ -8,3 +15,30 @@ type Results struct {
 	duration             float64 // Benchmark duration.
 	requestsPerSecond    int     // Cumber of requests per second - the main measured value.
 }
+
+// Request describes various types of requests.
+type Request struct {
+	operation             func(Request) // insertOperation, selectOperation or updateOperation.
+	ctx                   context.BenchCtx
+	tarantoolConnection   *tarantool.Connection
+	results               *Results
+	getRandomTupleCommand *string // for selectOperation or updateOperation.
+}
+
+// RequestsGenerator data structure for abstraction of a renewable heap of identical requests.
+type RequestsGenerator struct {
+	request Request // Request with specified operation.
```

Comment:
It may be more convenient to store the pointer instead of the value. Up to you.

Reply:
If I store the value, I don't need to care about object lifetime. In addition, the object itself needs only one, for each RequestSequence